web/src/main/java/model/User.java

```diff
@@ -0,0 +1,67 @@
+package model;
+
+
+import jakarta.persistence.*;
+import jakarta.transaction.Transaction;
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+@Entity
+@Table
+public class User {
+    @Id
```

Comment:
Always add new line after opening curly brackets. Whether that's a simple class, method, if condition, for loop ...